src/svgcontext.ts

```diff
@@ -91,13 +91,7 @@ class MeasureTextCache {
     const bbox = txt.getBBox();
     svg.removeChild(txt);
 
-    const fontSizeInPt: string = attributes['font-size'];
-
-    // Remove the trailing 'pt' from the font size and scale to convert from points to canvas pixel units.
-    // CSS specifies dpi to be 96 and there are 72 points to an inch: 96/72 == 4/3.
-    const height = Font.convertSizeToPixelValue(fontSizeInPt);
-
```

Comment:
What was the reasoning behind removing this chunk of code?

I ported this code in my Fonts PR, but I did not write the original "algorithm".... It is possible that the original author realized that the SVG bounding box was too high. It seems like the approach just uses the font size as a proxy, and turns it into a value in pixel units.

Reply:
This one was getting a fixed height rather than returning the ones calculated by `getBBox`